### PR TITLE
Add @MovieStoreGuy as CODEOWNER for Kafka receiver and exporter

### PR DESCRIPTION
**Pull request by mx-psi**
**Description:** 

Add @MovieStoreGuy as CODEOWNER for Kafka receiver and exporter.

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/3870#issuecomment-993169518

### CODE REVIEW

**Review by jpkrohling — APPROVED** [review]

**Comment by MovieStoreGuy** [comment]
I can't approve it after the fact but thank you :)

**Pull request merged**
